crates/wash-lib/src/start/wadm.rs

```diff
@@ -74,14 +74,25 @@ where
         // Check version to see if we need to download new one
         if let Ok(output) = Command::new(&wadm_bin_path).arg("--version").output().await {
             let stdout = String::from_utf8_lossy(&output.stdout).to_string();
-            if stdout.replace("wadm", "").trim() == version.trim_start_matches('v') {
+            print!("👀 Found wadm version: {}", stdout.trim_end());
+            let re = regex::Regex::new(r"^wadm[^\s]*").unwrap();     
+            if re.replace(&stdout, "").to_string().trim() == version.trim_start_matches('v') {
                 // wadm already exists, return early
+                println!(" Using it.✅");
```

Comment:
It might be nice to make this a bit more explicit -- `"using wadm version [{}]"` or something like that?

And could we keep the emoji location consistent here -- emoji at the front?

Reply:
Done with emoji, also separated work into 2 commits

Reply:
Thanks!